main.go

```diff
@@ -71,10 +72,25 @@ func printLog(data BeaconLog) {
 	}
 }
 
+// loadConfig loads the .beaconrc file into the beaconConfig
+func loadConfig(config *beaconConfig) error {
+	configFile, err := ioutil.ReadFile("./.beaconrc")
+	if err != nil {
+		return err
+	}
+
+	return json.Unmarshal(configFile, &config)
+}
+
 // ============================
 // MAIN!
 // ============================
 func main() {
+	var config beaconConfig
+
+	if err := loadConfig(&config); err != nil {
+		log.Fatal(err)
```

Comment:
I don't know what this does offhand (although I can guess), but I'm looking forward to learning more. I suspect we will need to implement this on the other file loading we're doing...